.buildkite/scripts/lifecycle/post_command.sh

```diff
@@ -52,4 +52,12 @@ if [[ "$IS_TEST_EXECUTION_STEP" == "true" ]]; then
     buildkite-agent artifact upload 'target/test_failures/**/*'
     ts-node .buildkite/scripts/lifecycle/annotate_test_failures.ts
   fi
+
 fi
+
+if [[ $BUILDKITE_COMMAND_EXIT_STATUS -ne 0 ]]; then
+  # If the slack team environment variable is set, ping the team in slack
+  if [ -n "${PING_SLACK_TEAM:-}" ]; then
+    ts-node .buildkite/scripts/lifecycle/ping_slack.ts
```

Comment:
Just a consideration: the PR is now mostly just wrapping around the typescript file, but that can be done with a simple one-liner
```
buildkite-agent meta-data set 'slack:ping_team:body' \
 "${PING_SLACK_TEAM}, can you please take a look at the test failures?"
``` 
(also add `ELASTIC_SLACK_NOTIFICATIONS_ENABLED == 'true'` above)

Reply:
@delanni, sure thing, I can simplify it. let me do it now

Reply:
Thanks! I prefer it like this, without the extra bloat for now (while this is not doing more than adding the metadata)